rel-eng/tito.props

```diff
@@ -462,6 +441,28 @@ whitelist = ansiblerole-foreman_scap_client
   rubygem-smart_proxy_remote_execution_ssh
   rubygem-smart_proxy_salt
   rubygem-smart_proxy_spacewalk
+  rubygem-systemu
+  rubygem-timeliness
+  rubygem-ttfunk
+  rubygem-vault
+  rubygem-wicked
+  rubygem-zscheduler
+
+[foreman-plugins-nightly-nonscl-rhel7]
+disttag = .el7
+whitelist = ansiblerole-foreman_scap_client
+  ansiblerole-insights-client
+  foreman-discovery-image-service
+  puppet-foreman_scap_client
+  rubygem-faraday
+  rubygem-fast_gettext
+  rubygem-mysql2
+  rubygem-netrc
+  rubygem-net-ssh
+  rubygem-newt
+  rubygem-openscap_parser
```

Comment:
no users at all currently

Reply:
I think they were adding this one to switch to it at some point. Not sure what to do with it right now.

Reply:
move it to scl-only and see if anyone yells?

Reply:
Done